static/routes.config.php

```diff
@@ -77,6 +78,7 @@
 			'/accounts/search'                   => [Module\Api\Mastodon\Accounts\Search::class,          [R::GET         ]],
 			'/accounts/update_credentials'       => [Module\Api\Mastodon\Accounts\UpdateCredentials::class, [R::PATCH     ]],
 			'/accounts/verify_credentials'       => [Module\Api\Mastodon\Accounts\VerifyCredentials::class, [R::GET       ]],
+			'/accounts/{name}'                   => [Module\Api\Mastodon\Accounts::class,                 [R::GET         ]],
```

Comment:
Did you verify the `/accounts/search` URL doesn't use the catch-all `/accounts/{name}` route?

Reply:
When I placed the new line some lines above, the router crashed with `Uncaught FastRoute\BadRouteException: Static route "/api/v1/accounts/relationships" is shadowed by previously defined variable route "/api/v1/accounts/([^/]+)" for method "GET"`. But this here seems to work.

Reply:
Also I verified that the whole functionality is working like expected.

Reply:
Excellent, thank you for making sure.